### PR TITLE
Add shouldafound documentation, introduce soft wrap for code blocks

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

### CODE REVIEW

**Review by spencerdrak — APPROVED** [review]

File: src/components/concept/_shouldafound.mdx [hunk 1]
```diff
@@ -0,0 +1 @@
+Directly report code which Semgrep did **not** match as you expected with the `shouldafound` option. The `shouldafound` enables you to report false negatives without leaving the comfort of your command-line interface (CLI).
```
Comment (s-santillan) [inline src/components/concept/_shouldafound.mdx:1]:
```suggestion
Directly report code which Semgrep did **not** match as you expected with the `shouldafound` option. The `shouldafound` enables you to report false negatives without leaving your command-line interface (CLI).
```

File: src/components/procedure/_shouldafound.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    ```sh
+    {
+        "email": "test@foo.com",
+        "lines": "func main() {\n    fmt.Println(\"foo\")\n}\n",
+        "message": "Semgrep missed vulnerable code here",
+        "path": "test.go"
+    }
+
+1. If you are fine with sending this information to r2c use `y`:
```
Comment (s-santillan) [inline src/components/procedure/_shouldafound.mdx:30]:
```suggestion
3. Press `y` to send this information to r2c:
```

File: src/css/custom.css [hunk 3]
```diff
@@ -181,6 +181,10 @@ dl {
     fill: currentColor;
 }
 
+code {
+  white-space : pre-wrap;
```
Comment (s-santillan) [inline src/css/custom.css:185]: This is a global change. Are we certain that nothing breaks? I'll pull this in tomorrow and check our config files, **but why are we doing this?**

**Comment by s-santillan** [comment]
The only reason I can't approve this (yet) is the inclusion of a global style to <code> tags, `white-space: pre-wrap;` without me knowing that all the other code blocks work as expected. I can do a quick check tomorrow but would still like to know why we are adding this attribute in.

**Comment by s-santillan** [comment]
Made a related thread on Slack about soft wrap. Opinions welcome.

shouldafound-softwrap
shouldafound-nowrap

File: src/css/custom.css (hunk 3 above)
Comment (adamkvitek) [reply src/css/custom.css:185]: @s-santillan so when you copy one of the code snippets (source code is here https://github.com/returntocorp/semgrep-docs/pull/669/files#diff-db99c4d68b8ef6118fce1fddd68ac32a10a9d1f5f03950e4f223921e152ff6f1R16-R18) it breaks in the terminal where you paste it unless we render it with the soft wrap. Feel free to remove this from the PR if we are breaking something.

File: src/css/custom.css [hunk 4]
```diff
@@ -181,6 +181,10 @@ dl {
...
+}
+
```
Comment (adamkvitek) [inline src/css/custom.css:187]:
```suggestion
```

**Pull request merged**